### ISSUE TITLE
User Embeddings Are Not Changed

### ISSUE THREAD

**Issue by mustfkeskin** [tensorflow/recommenders#454]
Hello

I train my custom data with retrieval model with 10 epoch with pre-trained embeddings.
I have products and users.
After model training completed I calculated cosine similarity with pre-trained embeddings and final model embeddings  for each embeddings

Sku embeddings average cosine similarity like 0.7 --> this is ok for me
User embeddings  average cosine similarity like 0.99 --> this is not

My model don't update my users embeddings. Where could i go wrong

**Comment by xiaoyaoyang** [comment]
did the training loss decrease while training?

**Comment by mustfkeskin** [comment]
yes it changes

it was due to my implementation error
thanks @xiaoyaoyang
